### PR TITLE
Move VTK cell type to pyvista.CellType

### PR DESCRIPTION
**Pull request by beroda**
Resolves https://github.com/pyvista/pyvista/issues/4440

### CODE REVIEW

File: tests/check_doctest_names.py
Comment (tkoyama010) [inline tests/check_doctest_names.py:14]: We can remove vtk import.
```suggestion
```

**Review by tkoyama010 — CHANGES_REQUESTED** [review]
Thanks working for this!

**Comment by beroda** [comment]
@tkoyama010 Glad to help a little

**Review by tkoyama010 — APPROVED** [review]

**Comment by tkoyama010** [comment]
Please hold off merging until we merge #4486 and resolve the conflict with it.

**Review by tkoyama010 — APPROVED** [review]

**Review by banesullivan — APPROVED** [review]

**Pull request merged**
